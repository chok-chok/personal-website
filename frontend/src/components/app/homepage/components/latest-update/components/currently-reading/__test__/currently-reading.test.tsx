import * as React from 'react';
import CurrentlyReading from '../currently-reading';
import { shallow } from 'enzyme';

describe('Test CurrentlyReading component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<CurrentlyReading />);
    expect(tree).toMatchSnapshot();
  });
});
