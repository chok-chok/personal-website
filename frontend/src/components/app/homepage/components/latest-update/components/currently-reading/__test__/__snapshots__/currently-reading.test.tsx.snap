// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CurrentlyReading component should renders correctly 1`] = `
<styled__CurrentlyReadingSection>
  <styled__HeaderWrapper>
    <styled__CurrentlyReadingHeader>
      Currently Reading
    </styled__CurrentlyReadingHeader>
    <styled__CurrentlyReadingLink
      href="/reading-list"
    >
      check my reading list
    </styled__CurrentlyReadingLink>
  </styled__HeaderWrapper>
  <styled__BookWrapper>
    <styled__BookImgWrapper>
      <styled__BookImg
        src="https:url"
      />
    </styled__BookImgWrapper>
    <styled__BookDetailWrapper>
      <styled__BookTitle>
        Agile Product Management
      </styled__BookTitle>
      <styled__AuthorInfo>
        by Eric Evans
      </styled__AuthorInfo>
    </styled__BookDetailWrapper>
  </styled__BookWrapper>
</styled__CurrentlyReadingSection>
`;
